firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -60,9 +61,9 @@
   // Whether the integration tests should run against a local Firestore emulator instead of the
   // Production environment. Note that the Android Emulator treats "10.0.2.2" as its host machine.
   // TODO(mrschmidt): Support multiple envrionments (Emulator, QA, Nightly, Production)
-  private static final boolean CONNECT_TO_EMULATOR = false;
+  private static final boolean CONNECT_TO_EMULATOR = BuildConfig.USE_EMULATOR_FOR_TESTS;
```

Comment:
Note this will not work internally, BuildConfig is not supported in google3

Reply:
FWIW, if we are blocked on this, we could replace this via Copybara. I assume it's ok to not run tests against the emulator in Google3.

Reply:
sg